src/thunderbots/software/network_input/networking/ssl_vision_client.cpp

```diff
@@ -46,13 +46,13 @@ void SSLVisionClient::handleDataReception(const boost::system::error_code& error
         // This function will be run for EVERY packet received from the network since the
         // last time poll() was called. This means that this function may be called
         // several times to process several packets. This is why we immediately store any
-        // received and processed data into the packet_queue. If we didn't save the
+        // received and processed data into the packet_vector. If we didn't save the
         // received data somewhere else, the data would be overwritten by the packet
         // being handled in the next handleDataReception function call
         auto packet_data = SSL_WrapperPacket();
         packet_data.ParseFromArray(raw_received_data_.data(),
                                    static_cast<int>(num_bytes_received));
-        packet_queue.emplace(packet_data);
+        packet_vector.emplace_back(packet_data);
```

Comment:
Hmm, why the switch over to `vector`? It was kinda nice to use a structure that explicitly preserves the order of insertion? Perhaps `std::deque` might be a better choice?

Reply:
Order doesn't _really_ matter here though. I can leave it as `queue`, `vector` was just nicer to iterator over.

Reply:
Ah yeah, I guess because all the packets are timestamped.

Reply:
There's also no guarantee (that I'm aware of) that we enqueue packets in the order they were received by the network. I'm not sure if the `handle...` functions are called in the order they were set up.